notebooks/kubeflow_pipelines/pipelines/solutions/kfp_pipeline_vertex_automl.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>may be worth mentioning here what is happening in the code below; i.e., we're granting storage bucket admin privledges to our default service account. I have a general question here about best IAM practices for granting this level of access (e.g. should one use a dedicated service account instead?) but I've seen/used/written this myself before so it's probably ok. </p><p> </p><p>Also, fwiw, i didn't find this step necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Removing this cell then, since this step is part of the general setup. </p>